freqtrade/pairlist/IPairList.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, freqtrade, config: dict) -> None:
         self._config = config
         self._whitelist = self._config['exchange']['pair_whitelist']
         self._blacklist = self._config['exchange'].get('pair_blacklist', [])
+        self._markets = self._freqtrade.exchange.get_markets()
```

Comment:
What's the reason for this? 
the only point `_markets` is used is to debug-log the whole markets-dict as list... ? 

also it's not refreshed, so you wouldn't see new/disabled markets

Reply:
This is a leftover from the changes to markets.. not necessary for this one